java/src/main/java/lesson01/README.md

```diff
@@ -113,9 +113,9 @@ import com.uber.jaeger.Configuration.ReporterConfiguration;
 import com.uber.jaeger.Configuration.SamplerConfiguration;
 
 public static com.uber.jaeger.Tracer initTracer(String service) {
-    SamplerConfiguration samplerConfig = new SamplerConfiguration("const", 1);
-    ReporterConfiguration reporterConfig = new ReporterConfiguration(true, null, null, null, null);
-    Configuration config = new Configuration(service, samplerConfig, reporterConfig);
+    SamplerConfiguration samplerConfig = new SamplerConfiguration().withType("const").withParam(1);
```

Comment:
@jpkrohling, I just started learning OpenTracing, so I'm not familiar with "export the relevant env vars", yet.

Reply:
Fair enough. This is particular to Jaeger (not generic OpenTracing), and it might indeed confuse users. For reference, I'm talking about `JAEGER_REPORTER_LOG_SPANS`, `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM`